PupDates-CrossPol/pupdates_frontend
# Sk testing

#### What's this PR do?

Completes basic testing for actions and some components including snapshots and mapState/mapDispatchToProps

#### Where should the reviewer start?

Wherever you see a test file.

#### How should this be manually tested?

Run ```npm i``` to get the dependencies installed, and then run ```npm run test``` to see what's passing (couple skipped currently) and see the coverage.

#### Any background context you want to provide?

Snapshots in React Native are weird. 

#### Questions:

Some of this will likely need to be updated as code implementing the rails backend is used and the response objects change what's in state. Also should possibly test apiCalls?
